@wtgodbe I think https://github.com/dotnet/aspnetcore/pull/22480/checks?check_run_id=732123555,

```
##[error]src\Servers\IIS\AspNetCoreModuleV2\AspNetCore\AspNetCore.vcxproj(316,5): error MSB3491: (NETCORE_ENGINEERING_TELEMETRY=Build) Could not write lines to file "F:\workspace\_work\1\s\artifacts\installers\Release\aspnetcoremodule.version". The process cannot access the file 'F:\workspace\_work\1\s\artifacts\installers\Release\aspnetcoremodule.version' because it is being used by another process.

```
 may be failing due to https://github.com/dotnet/aspnetcore/commit/bcd4fc05985c5a10505c14a428264dfab01c2632. I don't think PR should affect that file.

--- a/src/Servers/IIS/build/Build.Settings
+++ b/src/Servers/IIS/build/Build.Settings
@@ -26,6 +26,7 @@
        <PrecompiledHeader>Use</PrecompiledHeader>
        <TreatWarningAsError Condition="'$(TreatWarningsAsErrors)' != ''">true</TreatWarningAsError>
        <SDLCheck>true</SDLCheck>
+       <ControlFlowGuard>Guard</ControlFlowGuard>
        <StringPooling>true</StringPooling>
      </ClCompile>
      <Link>

